frontend/app/services/zuora/ZuoraActions.scala

```diff
@@ -139,6 +139,7 @@ case class Subscribe(memberId: MemberId, customerOpt: Option[Stripe.Customer], r
           <ns1:Subscription xsi:type="ns2:Subscription">
             <ns2:AutoRenew>true</ns2:AutoRenew>
             <ns2:ContractEffectiveDate>{now}</ns2:ContractEffectiveDate>
+            <ns2:ContractAcceptanceDate>{now}</ns2:ContractAcceptanceDate>
```

Comment:
For (my own, mostly) reference, I'm just going to annotate these lines with the Zuora documentation:

https://knowledgecenter.zuora.com/BC_Developers/SOAP_API/E1_SOAP_API_Object_Reference/Subscription

![image](https://cloud.githubusercontent.com/assets/52038/6639679/73b6923c-c989-11e4-88da-43f78ac5e872.png)